.github/workflows/test.yml

```diff
@@ -41,26 +40,25 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
 
       - name: Install Dependencies ⬆️
-        run: |
-          npm ci --no-optional
-          npx lerna bootstrap
+        run: yarn install --immutable
```

Comment:
`--immutable` tells yarn to blow up if there would be any changes made to the yarn.lock file. This would happen if the package.json somehow got out of sync with the yarn.lock. The previous `npm ci` command would also do that (among other things).